src/blocks/backlight.rs

```diff
@@ -0,0 +1,333 @@
+//! A block for displaying the brightness of a backlit device.
+//!
+//! This module contains the [`Backlight`](./struct.Backlight.html) block, which
+//! can display the brightness level of physical backlit devices. Brightness
+//! levels are read from and written to the `sysfs` filesystem, so this block
+//! does not depend on `xrandr` (and thus it works on Wayland). To set
+//! brightness levels using `xrandr`, see the
+//! [`Xrandr`](../xrandr/struct.Xrandr.html) block.
+
+use std::cmp::max;
+use std::convert::TryInto;
+use std::ops::Range;
+use std::path::{Path, PathBuf};
+
+use dbus::ffidisp::Connection;
```

Comment:
dbus supports async now

https://github.com/MaxVerevkin/swaystatus/blob/d0c04aa78487d66ae52f250cde61ecc73dd9da29/src/blocks/music.rs#L58-L66

https://github.com/MaxVerevkin/swaystatus/blob/d0c04aa78487d66ae52f250cde61ecc73dd9da29/src/blocks/music.rs#L125-L126

Reply:
> https://github.com/MaxVerevkin/swaystatus/blob/d0c04aa78487d66ae52f250cde61ecc73dd9da29/src/blocks/music.rs#L61-L66

That's what I missed, thanks!